### ISSUE TITLE
Update Docs for actions and how to work with this

### ISSUE THREAD

**Issue by oke-aditya** [oke-aditya/template_python#12]
## 📓 Documentation

New people might not know how CI and such actions work to ease their jobs.

Also, we need to specify where people need to edit a bit properly

**Comment by Gau02** [comment]
> ## 📓 Documentation
> New people might not know how CI and such actions work to ease their jobs.
> 
> Also, we need to specify where people need to edit a bit properly

@oke-aditya, Do you want to create a `README` which instructs how to check/use/edit CI workflows present in this repository?

**Comment by oke-aditya** [comment]
Yes. I want documentation about how these CIs run and what they do.

**Comment by oke-aditya** [comment]
That readme can go via mkdocs to this website docs. It has to be md file not rst.
